.circleci/config.yml

```
@@ -111,11 +111,27 @@ jobs:
           command: yarn
       - run:
           name: Install Pods
-          command: cd ios && pod install
+          working_directory: ios
+          command: pod install
       - run:
           name: Build and run tests
-          command: cd ios && xcodebuild  -allowProvisioningUpdates   -workspace InstabugSample.xcworkspace    -scheme InstabugSample    -sdk iphonesimulator    -destination 'platform=iOS Simulator,name=iPhone 13 Pro Max,OS=15.5'    test | xcpretty
+          working_directory: ios
+          command: xcodebuild -allowProvisioningUpdates -workspace InstabugSample.xcworkspace -scheme InstabugSample -resultBundlePath coverage/result.xcresult -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 13 Pro Max,OS=15.5' test | xcpretty
```

I think it's better if we break this into multiple lines.

```suggestion
          command: |
                   xcodebuild -allowProvisioningUpdates \
                       -workspace InstabugSample.xcworkspace \
                       -scheme InstabugSample \
                       -resultBundlePath coverage/result.xcresult
                       -sdk iphonesimulator \
                       -destination 'platform=iOS Simulator,name=iPhone 13 Pro Max,OS=15.5' \
                       test | xcpretty
```